webui/react/src/components/ResourcePoolCard.module.scss

```diff
@@ -46,7 +45,7 @@ $font-size-content: 12px;
     }
     .default {
       color: var(--theme-surface-on);
-      display: flex;
+      margin-left: 8px;
```

Comment:
This moves the "default" label to the left to have space for action menu.
<img width="327" alt="Screenshot 2023-07-05 at 12 56 18 PM" src="https://github.com/determined-ai/determined/assets/40620519/1861d4a4-425f-4d86-ad09-9726fb7c1632">